app/views/reminder_mailings/show.html.slim

```diff
@@ -0,0 +1,53 @@
+= render 'section_navigation'
+
+h1 #{@reminder_mailing.t_enum(:kind)} Erinnerungs-Email
+
+h2 Versand Einzelheiten
+
+dl.dl-horizontal
+  dt Art
+  dd= @reminder_mailing.t_enum(:kind)
+  dt Erstellt am
+  dd= l(@reminder_mailing.created_at.to_date)
+  dt Status
+  - if @reminder_mailing.sending_triggered
+    dd Übermittelt
+  - else
+    dd Nicht versandt
```

Comment:
for enhancing readability this view, since a lot of information are shown, maybe a `hr` would be helpful here